tpcx_bb/queries/q02/tpcx_bb_query_02_sql.py

```diff
@@ -44,8 +44,8 @@ def main(data_dir, client, bc, config):
 
     query_1 = """
         SELECT
-            wcs_user_sk,
-            wcs_item_sk,
+            CAST( wcs_user_sk AS INTEGER) AS wcs_user_sk,
```

Comment:
Is this downcasting to int32 from int64?

Reply:
Yes.